apps/bmd/script/vvsg-cloc

```diff
@@ -151,12 +155,12 @@ function formatCLOCStat(minCLOC, actual, expected) {
     `${minCLOC.toString().padStart(3, ' ')}+ CLOC: ${Math.round(actual * 100)
       .toString()
       .padStart(2, ' ')}% (<= ${Math.round(
-        expected * 100
-      ).toString()}% expected)`
+      expected * 100
+    ).toString()}% expected)`
   )
 }
 
-main().catch(error => {
```

Comment:
These other changes are because this file isn't properly set up to be auto-formatted by prettier.